float32 solve_time
---
float32 previous_time
float32 new_time
